differ/storage.go

```diff
@@ -547,10 +549,10 @@ func (storage DBStorage) ReadLastNotifiedRecordForClusterList(clusterEntries []t
 		clusterIDs[idx] = string(entry.ClusterName)
 	}
 
-	whereClause := fmt.Sprintf(` WHERE org_id IN (%v) AND cluster IN (%v) AND state = 1 `,
+	whereClause := fmt.Sprintf(` WHERE org_id IN (%v) AND cluster IN (%v) AND event_type_id = 1 AND state = 1`,
```

Comment:
very probably it will be faster to switch the WHERE conditions and put event_type and state on 1st place